extensions/betterdiscord-search/src/index.tsx

```diff
@@ -0,0 +1,207 @@
+  const cancelRef = useRef<AbortController | null>(null);
+
+  useEffect(() => {
+    search("");
+    return () => {
+      cancelRef.current?.abort();
+    };
+  }, []);
+
+  async function search(searchText: string) {
+    cancelRef.current?.abort();
+    cancelRef.current = new AbortController();
+    try {
+      setState((oldState) => ({
+        ...oldState,
+        isLoading: true,
+      }));
+      const results = await performSearch(searchText, cancelRef.current.signal);
+      setState((oldState) => ({
+        ...oldState,
+        results: results,
+        isLoading: false,
+      }));
+    } catch (error) {
+      if (error instanceof AbortError) {
+        return;
+      }
+      mainToast.style = Toast.Style.Failure;
+      mainToast.title = "Failed to Perform Search";
+      mainToast.message = `${error}`;
+    }
+  }
+
+  return {
+    state: state,
+    search: search,
+  };
+}
+
+export default function Command() {
+  const { push, pop } = useNavigation();
+  const { state, search } = useSearch();
+  return (
+    <List isLoading={state.isLoading} searchBarPlaceholder="Search for a BetterDiscord Addon..." throttle>
+      <List.Section title="Addons" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult) => (
+          <SearchListItem key={searchResult.id} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+
+  function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+    const details = `
+# ${searchResult.name}
+#### By ${searchResult.author} | Version: ${searchResult.version} | Downloads: ${searchResult.downloads} | Likes: ${searchResult.likes}
+
+${searchResult.description}
+
+<img src="${searchResult.icon}" alt="Thumbnail"/> 
+  `;
+    mainToast.hide();
+    return (
+      <List.Item
+        title={searchResult.name}
+        icon={{
+          source: searchResult.thumb || Icon.QuestionMark,
+          tintColor: "white",
+        }}
+        accessoryTitle={searchResult.author}
+        actions={
+          <ActionPanel>
+            <Action.OpenInBrowser url={searchResult.url} />
+            <Action
+              title="Details"
+              onAction={() => {
+                push(<Details searchResult={searchResult} description={details} />);
+              }}
+            />
+          </ActionPanel>
+        }
+      />
+    );
+  }
+  function Details(props: { description: string; searchResult: SearchResult }) {
+    const { pop } = useNavigation();
+
+    return (
+      <Detail
+        navigationTitle="Details"
+        markdown={props.description}
+        actions={
+          <ActionPanel title="Detail">
+            <Action title="Back" onAction={pop} />
+          </ActionPanel>
+        }
+      />
+    );
+  }
+}
+
+async function performSearch(searchText: string, signal: AbortSignal): Promise<SearchResult[]> {
+  const params = new URLSearchParams();
```

Comment:
```suggestion
  const params = new URLSearchParams();
```
Leftover or forgotten?